Migration.md

```diff
@@ -0,0 +1,351 @@
+
+```typescript
+const UsVerifications = new USVerificationsApi(av_config);
+const verificationData1: UsVerificationsWritable = {
+  primary_line: "001 CEMETERY LANE",
+  city: "WESTFIELD",
+  state: "NJ",
+  zip_code: "07090",
+};
+
+const verificationData2: UsVerificationsWritable = {
+  primary_line: "1515 CEMETERY LN",
+  city: "WESTFIELD",
+  state: "NJ",
+  zip_code: "07090",
+};
+const addressList: MultipleComponentsList = {
+  addresses: [verificationData1, verificationData2],
+};
+
+try {
+  const bulkVerified = await UsVerifications.verifyBulk(addressList);
+} catch (err: any) {
+  console.error(err);
+}
+```
+
+​Again, as mentioned above, If you are using the new TypeScript SDK using JavaScript,the code is essentially the same apart from removing the type identifiers found in TypeScript This pattern may be followed for all methods where examples are not supplied as shown here:
+
+```javascript
+const UsVerifications = new USVerificationsApi(av_config);
+const verificationData1= {
+    primary_line: "001 CEMETERY LANE",
+    city: "WESTFIELD",
+    state: "NJ",
+    zip_code: "07090"
+};
+const verificationData2= {
+    primary_line: "1515 CEMETERY LN",
+    city: "WESTFIELD",
+    state: "NJ",
+    zip_code: "07090",
+};
+const addressList={
+    addresses: [verificationData1, verificationData2]
+}
+
+try {
+  const bulkVerified = await UsVerifications.verifyBulk(addressList);
+} catch (err: any) {
+       console.error(err);
+}
+```
+
+### COMPARE SINGLE ADDRESS VERIFICATION METHOD
+
+​Here is a sample of the lob-node SINGLE VERIFY method:
+
+```javascript
+​Lob.usVerifications.verify({
+  primary_line: '1313 CEMETERY LN',
+  city: 'WESTFIELD',
+  state: 'NJ',
+  zip_code: '07000'
+}, function (err, res) {
+  console.log(err, res);
+});
+```
+
+​Here is a sample of the TypeScript SDK Single Verify method:
+
+```typescript
+const UsVerifications = new USVerificationsApi(av_config);
+const verificationData1: UsVerificationsWritable = {
+  primary_line: "001 CEMETERY LANE",
+  city: "WESTFIELD",
+  state: "NJ",
+  zip_code: "07090",
+};
+
+try {
+  const singleVerified = await UsVerifications.verifySingle(verificationData1);
+} catch (err: any) {
+  console.error(err);
+}
+```
+
+### COMPARE BANK ACCOUNT VERIFY
+
+​Here is a sample of the lob-node Bank Account Verify method:
+
+```javascript
+Lob.bankAccounts.verify(
+  "bank_dfceb4a2a05b57e",
+  {
+    amounts: [25, 63],
+  },
+  function (err, res) {
+    console.log(err, res);
+  }
+);
+```
+
+Here is a sample of the TypeScript SDK Bank Account Verify method:
+
+```typescript
+const verificationData= new BankAccountVerify({
+    amounts: [11, 35],
+});
+
+const bankData= new BankAccountWritable = ({
+    description: "Test Bank Account",
+    routing_number: "322271627",
+    account_number: "123456789",
+    signatory: "Gomez Addams",
+    account_type: BankTypeEnum.Individual,
```

Comment:
remove trailing comma (I see several other object that have trailing commas after the last attribute. My preference is to standardize on not using it